PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -340,6 +340,7 @@ DECLARE_SOA_COLUMN(PtHadron, ptHadron, float);                             //! T
 DECLARE_SOA_COLUMN(MD, mD, float);                                         //! Invariant mass of D+
 DECLARE_SOA_COLUMN(MlScoreBkg, mlScoreBkg, float);                         //! ML background score for D+ selection
 DECLARE_SOA_COLUMN(MlScorePrompt, mlScorePrompt, float);                   //! ML prompt score for D+ selection
+DECLARE_SOA_COLUMN(MlScoreFd, mlScoreFd, float);                           //! ML FD score for D+ selection
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(MlScoreNonPrompt, mlScoreNonPrompt, float);                           //! ML non-prompt score for D+ selection
```

Reply:
Thanks. I will do it.  There is one warning magic number, and this comes from three scores read in the classes.
The only way I can solve is to define variables and assign them names (outputMl[0], outputMl[1], outputMl[2]).
This will include an extra step and can make the code slower. Should I ignore them?

Reply:
Why would you define variables? What I see are comparisons to origin constants and one comparison to a number of daughters which is also a compile-time constant.

Reply:
I think the documentation was not clear to me, as written in the link below. I will fix them.
https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html#Magic_numbers

Reply:
Hi Vit, I got the trick on how to identify the cause of errors and fix them. I am fixing all the errors.